geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js

```diff
@@ -253,15 +253,53 @@ Canvas.defaultProps = {
     reset: false,
     autorotate: false,
     wireframe: false,
+    zoomTo: [],
+    rotationSpeed: 0.5,
+    movieFilter: false,
     depthWrite: true,
-    zoomTo: undefined,
+    position: {
+      x: 319.7,
+      y: 153.12,
+      z: -494.2,
+    },
+    rotation: {
+      rx: -3.14,
+      ry: 0,
+      rz: -3.14,
+      radius: 559.83,
+    },
     cameraControls: {
+      instance: null,
```

Comment:
Default to null doesn't seem to work. I think we should make the cameraControls component provided by geppetto, the default cameraControls.instance of the canvas.

Reply:
Is this the cameraControls you are talking about? https://github.com/MetaCell/geppetto-meta/blob/development/geppetto.js/geppetto-ui/src/camera-controls/CameraControls.js

Reply:
Would it look like this? `instance: new CameraControls()`

Reply:
Yes for the first question.
No to the second. It would look like [this](https://github.com/MetaCell/geppetto-NeuroSCAN/blob/develop/applications/neuroscan/frontend/src/utilities/functions.js#L58-L59):
```
cameraControls: {
          instance: CameraControls,
          props: {},
        },
```